src/metabase/api/search.clj

```diff
@@ -89,7 +89,9 @@
    :database_id         :integer
    :table_schema        :text
    :table_name          :text
-   :table_description   :text))
+   :table_description   :text
+   ;; returned for Database and Table
+   :initial_sync_status :text))
```

Comment:
Not sure if I'm totally understanding all of the search code correctly. Please let me know if this is not the correct way to add a field to the models returned by `/api/search` 🙏

Reply:
I'll look into it but its likely that we want to totally ignore these properties in search. We don't search on them and they are not some core property of dbs or tables. My guess is we want to omit it completely from all of this. Unless there is some desire to show sync status of tables returned from search results. But my guess is that we don't care?

Reply:
We do want to show the sync status for databases & tables undergoing their initial sync in search, [per the product doc](https://www.notion.so/metabase/Provide-progress-on-initial-database-sync-16af8e79312a45ecb715d8e58801c52b#0c54800c0a9e4dd7894128ca6e8e35f6).

However we don't want to search on these fields, just return them in the API response for the relevant models.